Doc/tools/extensions/pyspecific.py

```diff
@@ -253,7 +253,7 @@ def run(self):
             text = 'The NEWS file is not available.'
             node = nodes.strong(text, text)
             return [node]
-        content = issue_re.sub(r'`\1ssue #\2 <https://bugs.python.org/\2>`__',
+        content = issue_re.sub(r'`\bpo-\2 <https://bugs.python.org/\2>`__',
```

Comment:
Sure, you can either you use `\1` or insert `bpo-`. But here you overwrote half the capture group by only deleting the `1` and not the `\`.